cli/.npmignore

```diff
@@ -1 +1,7 @@
-bin/cargo-generate
\ No newline at end of file
+
```

Comment:
I would prefer to ignore bin/proving-keys folder instead of files inside (this list of files is not comprehensive, if you download all proving keys via `/light-prover/scripts/dowload_keys.sh full`, this list would be a much longer.